Formula/apache-spark.rb

```diff
@@ -18,6 +18,6 @@ def install
   end
 
   test do
-    system "#{bin}/spark-shell <<<'sc.parallelize(1 to 1000).count()'"
+    system "echo 'sc.parallelize(1 to 1000).count()' | #{bin}/spark-shell"
```

Comment:
I tried different variations, but pipe_output doesn't send command "sc.parallelize(1 to 1000).count()"  inside the prompt of spark-shell. 

When you run spark-shell, the prompt scala> appears and function "sc.parallelize(1 to 1000).count()" should be entered there. 

Any suggestions please?